quora-service/src/main/java/com/upgrad/quora/service/business/PasswordCryptographyProvider.java

```diff
@@ -27,7 +27,7 @@
      * @param password char array.
      * @return String array with [0] encoded salt [1] hashed password.
      */
-    public String[] encrypt(final String password) {
+    public static String[] encrypt(final String password) {
```

Comment:
Any reason behind changing this to static ?

Reply:
There were compilation issues because we have called this method without creating object of the class.

Reply:
ok